l1-contracts/src/interfaces/IUniFiAVSManager.sol

```diff
@@ -232,6 +266,13 @@ interface IUniFiAVSManager {
      */
     function setDeregistrationDelay(uint64 newDelay) external;
 
+    /**
+     * @notice Sets a new registration delay for validators.
+     * @param newDelay The new registration delay in seconds.
+     * @dev Restricted to the DAO
+     */
+    function setRegistrationDelay(uint64 newDelay) external;
```

Comment:
IMO interface doesn't need to contain functions that are not meant to to be called by public. It adds unnecessary noise and LOC. I vote fore removing functions like this from the Interface and just keeping them + natspec in the contract

Reply:
True, but DAO contracts generally do not import interfaces. They deal with target & calldata. e.g. https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/Governor.sol